modules/puppet_proxy/dependency_injection/dependencies.rb

```diff
@@ -1,45 +1,46 @@
-require 'puppet_proxy/runtime_configuration'
-
+# rubocop:disable Style/Lambda
```

Comment:
Please keep this enabled globally, disabling it per-file makes the codebase inconsistent.